migrations/20200325204238-mv-data-from-accounts-to-recoverymethods.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+module.exports = {
+    up: queryInterface => {
+        return queryInterface.sequelize.transaction(transaction => {
+            return new Promise(resolve => {
+                Promise.all([
+                    queryInterface.sequelize.query(
+                        'SELECT * FROM "Accounts" WHERE "phoneNumber" IS NOT NULL',
+                        { transaction }
+                    ),
+                    queryInterface.sequelize.query(
+                        'SELECT * FROM "Accounts" WHERE "email" IS NOT NULL',
+                        { transaction }
+                    )
+                ]).then(([[phoneRecords], [emailRecords]]) => {
+                    Promise.all([
+                        phoneRecords.length && queryInterface.bulkInsert(
```

Comment:
we have a small DB, so this might work (test with Render snapshot) – but generally it's best to run raw SQL for bulk ops like this vs loading all rows to RAM

Reply:
I would prefer to do that but wasn't sure if Sequelize gave me the tools. I guess raw SQL would have been the way. I'll test this and see if it's acceptably performant.

Reply:
given that this is one off, if it works on dump – no point in optimizing, it's more just for future cases